### PR TITLE
Remove documentation warnings.

### PR DESCRIPTION
**Pull request by Natim**
r? @leplatrem

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -65,6 +65,6 @@ maintainer-clean: distclean
 # 	  EXIT_CODE=$$?; kill $$PID; exit $$EXIT_CODE
 
 docs: install-dev
-	$(VENV)/bin/sphinx-build -b html -d $(SPHINX_BUILDDIR)/doctrees docs $(SPHINX_BUILDDIR)/html
+	$(VENV)/bin/sphinx-build -a -n -b html -d $(SPHINX_BUILDDIR)/doctrees docs $(SPHINX_BUILDDIR)/html
```
Comment (leplatrem) [inline Makefile:68]: If you have no warning at all, you can even specify `-W` that will fail the build on warning :)

**Comment by leplatrem** [comment]
:+1: r+

File: Makefile (hunk 1 above)
Comment (Natim) [inline Makefile:68]: Will try.

Comment (Natim) [inline Makefile:68]: WARNING: nonlocal image URI found: https://travis-ci.org/mozilla-services/syncto.svg?branch=master

**Pull request merged**